libraries/AC_Avoidance/AP_OAPathPlanner.h

```diff
@@ -93,6 +102,8 @@ class AP_OAPathPlanner {
     AP_Float _lookahead;            // object avoidance will look this many meters ahead of vehicle
     AP_Float _margin_max;           // object avoidance will ignore objects more than this many meters from vehicle
     AP_Int16 _options;              // Bitmask for options while recovering from Object Avoidance
+    AP_Int16 _bendy_type;           // Type of BendyRuler to run
```

Comment:
let's make this an AP_Int8

Reply:
bitmask? Or maybe there is a plan to add 3D as a new type

Reply:
@IamPete1, I don't think we are planning to allow multiple bendy-ruler algorithms to run at the same time so just a single type should be ok.